Cargo.toml

```diff
@@ -14,6 +15,7 @@ authors = ["Concordium <developers@concordium.com>"]
 [workspace.dependencies]
 anyhow = "1.0.99"
 async-trait = "0.1.89"
+axum = { version = "0.8.4", features = ["http2"] }
```

Comment:
So I'm guessing even though only one part of the workspace (the service) will use axum, the convention is to add everything to the workspace dependencies unless you have a good reason not to, e.g. needing different versions of the same dependency in different crates of the workspace?